util-core/src/main/scala/com/twitter/util/BatchExecutor.scala

```diff
@@ -22,9 +22,8 @@ import scala.collection.mutable
  * - Rather than having separate sizeThreshold and sizePercentile parameters,
  *   just have a single call-by-name sizeThreshold parameter, and let the
  *   caller implement whatever logic they want to compute the next batch size.
- * - Return an instance of a new class (Batcher?) which extends
- *   `In => Future[Out]`. Could support things like querying the queue size,
- *   forcing a flush, attaching callbacks to flush operations, etc.
+ * - Add more functionality to class Batcher. Could support things like querying the queue size,
```

Comment:
this doesn't really belong in scaladoc.  Could you move it out into a `// TODO` comment?